applications/msai.md

```diff
@@ -0,0 +1,290 @@
+|-------:|----------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                                | Apache 2.0                                                                                                                                                                                                                                                                                                                               |
+|    0b. | Documentation                          | The extended API will be documented                                                                                                                                                                                                                                                                                                      |
+|    0c. | Testing Guide                          | Each epic describes testing opportunities.  [ 1. Create a health blockchain](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) 2. Create openapi specifcations for reading and writing data to and from the NFT smart contract (Depends on blockchain nft tech used) .                                             |
+|    0d. | Docker                                 | A dockerized setup of the parachain will be provided after several iterations when we have a working secure application against a customised subtrate blockchain with extended off chain worker local storage                                                                                                                            |
+|    0e. | Article                                | Press release will be prepared a few weeks before the official rollout of the network. The link to the article will be sent when ready                                                                                                                                                                                                   | 
+|    1a. | Node Repo                              | Complete the deployment of the basic public chain with example  storage of a native DB format through the off chain worker local storage API.                                                                                                                                                                                            |
```

Comment:
Please specify what exactly you mean by "storage of a native DB format through the off chain worker local storage API".

Reply:
What this referred to is the original generic data format tables described int the ### Project Details in the document. However I have since updated this #nilstone. I will go through and try  to clarify all individual steps.  
Off chain worker api allows A single Node to store some data off chain... What we need here is data replication and data format/query capability for the type of the data described.